ts/screens/showroom/core/IllustrationsShowroom.tsx

```diff
@@ -0,0 +1,183 @@
+import PlacingCard from "../../../../img/cie/place-card-illustration.png";
+
+const illustrationItemGutter = 16;
+
+const styles = StyleSheet.create({
+  itemsWrapper: {
+    flexDirection: "row",
+    flexWrap: "wrap",
+    justifyContent: "flex-start",
+    marginLeft: (illustrationItemGutter / 2) * -1,
+    marginRight: (illustrationItemGutter / 2) * -1,
+    marginBottom: 16
+  },
+  illustrationWrapper: {
+    width: "50%",
+    justifyContent: "flex-start",
+    marginBottom: 16,
+    paddingHorizontal: illustrationItemGutter / 2
+  },
+  bgImage: {
+    position: "absolute",
+    width: "150%",
+    height: "150%"
+  },
+  image: {
+    width: "100%",
+    height: "100%"
+  },
+  illustrationItem: {
+    overflow: "hidden",
+    position: "relative",
+    aspectRatio: 1,
+    borderRadius: 8,
+    padding: 24,
+    alignItems: "center",
+    justifyContent: "center",
+    borderColor: "rgba(0, 0, 0, 0.1)",
+    borderWidth: 1
+  },
+  rasterPill: {
+    position: "absolute",
+    right: 8,
+    top: 8,
+    overflow: "hidden",
+    backgroundColor: IOColors.orange,
+    color: "#FFFFFF",
+    fontSize: 9,
+    textTransform: "uppercase",
+    padding: 4,
+    borderRadius: 4
+  }
+});
+
+export const IllustrationsShowroom = () => (
+  <ShowroomSection title={"Illustrations"}>
+    <H2 color={"bluegrey"} weight={"SemiBold"} style={{ marginBottom: 16 }}>
+      Onboarding
+    </H2>
+    <View style={styles.itemsWrapper}>
+      <IllustrationBox
+        raster={true}
+        name={"05"}
+        image={renderRasterImage(Landing05)}
+      ></IllustrationBox>
+      <IllustrationBox
+        raster={true}
+        name={"01"}
+        image={renderRasterImage(Landing01)}
+      ></IllustrationBox>
+      <IllustrationBox
+        raster={true}
+        name={"02"}
+        image={renderRasterImage(Landing02)}
+      ></IllustrationBox>
+      <IllustrationBox
+        raster={true}
+        name={"03"}
+        image={renderRasterImage(Landing03)}
+      ></IllustrationBox>
+      <IllustrationBox
+        raster={true}
+        name={"04"}
+        image={renderRasterImage(Landing04)}
+      ></IllustrationBox>
+    </View>
+    <H2 color={"bluegrey"} weight={"SemiBold"} style={{ marginBottom: 16 }}>
+      CIE
+    </H2>
+    <View style={styles.itemsWrapper}>
+      <IllustrationBox
+        raster={true}
+        name={"CIE"}
+        image={renderRasterImage(LandingCIE)}
+      ></IllustrationBox>
+      <IllustrationBox
+        raster={true}
+        name={"Placing Card"}
+        image={renderRasterImage(PlacingCard)}
+      ></IllustrationBox>
+    </View>
+  </ShowroomSection>
+);
+
+type IllustrationBoxProps = {
+  name: string;
+  image: React.ReactNode;
+  raster?: boolean;
+};
+
+const renderRasterImage = (image: ImageSourcePropType) => (
+  <NativeImage
+    source={image}
+    resizeMode={"contain"}
+    style={styles.image}
+    testID={"rasterImage"}
+  />
+);
+
+const IllustrationBox = (props: IllustrationBoxProps) => (
```

Comment:
What's the difference between `IllustrationBox` and `PictogramBox` can both components be summarized into one common and standalone component?

Reply:
You're right. I needed something quick and dirty to view both assets at first. Now I've added a new component AssetViewerBox that replaces both.